step-by-step/04-check-compat.md

```diff
@@ -1,24 +1,20 @@
 # 4. Checking compatibility
 
-We are going to check if this device is supported.
+We are going to check if your device is supported.
 
-1. launch RVM. Ignore any updates it claimed it knew.
+1. Launch ReVanced Manager. Ignore any updates it claimed it knew.
 ==- Image for reference
 ![launch rvm](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/040-first_launch_manager.jpg?raw=true)
 ===
 2. Go to the settings tab and scroll down until you find "About"
 ==- Image for reference
 ![check about](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/050-check_about.jpg?raw=true)
 ===
-3. Check that `Arch` had at least 1 entry that looks like `arm64-v8a`
-4. If you only got something that said `armv7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-5. If you got `armv8` or higher, you are probably fine
-6. Check that your Android version is at least 8
-7. If you only got something that said `7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-8. If you made it here, you can probably patch on your phone with ReVanced manager
+3. Check that `Arch` says `arm64-v8a`
+4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armv7` not supported?
+==- What does `arch` mean? Why is `armeabi-v7a` not supported?
```

Comment:
`armeabi-v7a` means emulated armv7 (32 bit). This appears on devices that has `armv8` but wanted to retain compatibility with older apks. 

`armv7` is the intended.